### ISSUE TITLE
Giving you feedback on Lazarus --os=linux --cpu=i386 for test_cwRuntime

### ISSUE THREAD

**Issue by gcarreno** [chapmanworld/cwRuntime#5]
Hi Craig,

I'm attaching the binary and the xml file with the result of running test_cwRuntime on Lazarus with `--os=linux` `--cpu=i386`

TLDR: Builds/Fails tests

Result file: [test_cwRuntime-linux-i386.xml.txt](https://github.com/chapmanworld/cwRuntime/files/4506032/test_cwRuntime-linux-i386.xml.txt)

Binary: [test_cwRuntime-linux-i386.zip](https://github.com/chapmanworld/cwRuntime/files/4506034/test_cwRuntime-linux-i386.zip)

I remembered reading in one of Don's (@LongDirtyAnimAlf) issues, that he cross-compiles all his versions of [fpcupdeluxe](https://github.com/LongDirtyAnimAlf/fpcupdeluxe) from one machine and decided to try linux-i386 just for the fun of it.

I've dusted the Raspberry Pi 2 and I'll commence the updatezzz ;)

Cheers,
Gus

**Comment by gcarreno** [comment]
Hey Craig.

As per commit 7d9a264 it is Building AND Passes tests.

Result File: [test_cwRuntime-linux-i386.xml.txt](https://github.com/chapmanworld/cwRuntime/files/4513621/test_cwRuntime-linux-i386.xml.txt)

Binary: [test_cwRuntime-linux-i386.zip](https://github.com/chapmanworld/cwRuntime/files/4513623/test_cwRuntime-linux-i386.zip)

Cheers,
Gus
